### PR TITLE
Add new debug commands to docs

### PR DESCRIPTION
**Pull request by treacherousfiend**
Add commands added by #510 to docs. This PR probably needs some adjustments, but its a good baseline.

### CODE REVIEW

**Comment by Tiagoquix** [comment]
https://github.com/treacherousfiend/mastercomfig/pull/5

@treacherousfiend docs changes only

**Comment by micwoj92** [comment]
@treacherousfiend

**Comment by Tiagoquix** [comment]
@treacherousfiend https://github.com/mastercomfig/mastercomfig/commit/42cde266ad89e337572c0dabfbf11b9c62b6ddac

debug_output_display should be added to the docs

File: docs/customization/custom_configs.md [hunk 1]
```diff
@@ -115,7 +115,8 @@ mastercomfig provides a set of handy debugging commands used during mastercomfig
 ### General
 
 * `debug_output`: Turn on developer only output (`debug_output_1`). Shows various warnings about potential issues, and outputs console to the corner of the screen.
-* `debug_output_toggle`: Cycle through all 4 modes for developer only output. Different modes display different amounts of information.
+* `debug_output_toggle`: Cycle through all 4 modes (including disabled) for developer only output. Different modes display different amounts of information.
+* `debug_output_display`: Display console output in the corner of the screen without adding additional information.
```
Comment (Tiagoquix) [inline docs/customization/custom_configs.md]: @treacherousfiend replace "adding" with "showing" or remove it

"adding additional information" is weird

**Review by Tiagoquix — CHANGES_REQUESTED** [review]

**Review by mastercoms — APPROVED** [review]

**Pull request merged**